kubeflow/fairing/frameworks/lightgbm_dist_training_init.py

```diff
@@ -1,9 +1,14 @@
+from kubeflow.fairing.constants import constants
+
 import importlib
 import argparse
 import logging
 
-logging.basicConfig(format='%(message)s')
-logging.getLogger().setLevel(logging.INFO)
+logging.basicConfig(
+    format=constants.FAIRING_LOG_FORMAT,
+    datefmt=constants.FAIRING_LOG_DATEFMT,
+)
+logging.getLogger().setLevel(constants.FAIRING_LOG_LEVEL)
```

Comment:
One of the things I noticed that setLevel generally takes a number value defined in log.py such as 
DEBUG = 1                                                                                                                                                                                                                                      
INFO = 2                                                                                                                                                                                                                                       
WARN = 3                                                                                                                                                                                                                                       
ERROR = 4                                                                                                                                                                                                                                      
FATAL = 5

Whereas with this change, it seems like it becomes a string value. Could that be causing issues?

Reply:
@xauthulei Is that possiable to set level in `basicConfig`? such as below?
```
logging.basicConfig(level=kubeflow.fairing.constants.FAIRING_LOG_LEVEL)
```

Reply:
@abhi-g , That works in my local, even if the level is `string value`, :
```
llhu@huleis-mbp fairing % python3 
Python 3.7.3 (default, Nov 15 2019, 04:04:52) 
[Clang 11.0.0 (clang-1100.0.33.16)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import logging
>>> logging.getLogger().setLevel(logging.INFO)
>>> logging.info("HI")
INFO:root:HI
>>> logging.getLogger().setLevel('ERROR')
>>> logging.info("HI")
>>> logging.critical("HI")
CRITICAL:root:HI
>>> logging.error("HI")
ERROR:root:HI
>>> logging.info("HI")
>>> 
```